### PR TITLE
Fix history.json api endpoint

### PR DESCRIPTION
**Pull request by vasilty**

### CODE REVIEW

File: rt_dashboard/templates/rt_dashboard/history.html [hunk 1]
```diff
@@ -23,8 +23,8 @@
 <script type="text/javascript">
 window.addEventListener('load', function() {
     var container = document.getElementById('visualization');
-    var items = new vis.DataSet([ {{rows}} ]);
-    var groups = [ {{groups}} ]
+    var items = new vis.DataSet(JSON.parse(JSON.stringify({{rows}})));
+    var groups = JSON.parse(JSON.stringify({{groups}}));
```
Comment (groupsky) [inline rt_dashboard/templates/rt_dashboard/history.html]: This shouldn't be needed - `JSON.parse(JSON.stringify(...))` just makes a clone of the data, but in this case it's not used anywhere else.

File: rt_dashboard/history.py [hunk 2]
```diff
@@ -114,17 +107,12 @@ def get_history():
         elif t.status == 'running':
             keys['style'] = 'border-color: {0}; background-color: {0}'.format('#D5F6D7')
 
-        keys = {k: jsdate(v) if isinstance(v, datetime.datetime) else repr(v)
+        keys = {k: v.isoformat() if isinstance(v, datetime.datetime) else v
```
Comment (groupsky) [inline rt_dashboard/history.py]: Can we use timestamp instead?

File: rt_dashboard/templates/rt_dashboard/history.html (hunk 1 above)
Comment (vasilty) [reply rt_dashboard/templates/rt_dashboard/history.html]: Template doesn't have access to the JSON string, but to the python data structures. So I'd argue we need this

File: rt_dashboard/history.py (hunk 2 above)
Comment (vasilty) [reply rt_dashboard/history.py]: sure

File: rt_dashboard/templates/rt_dashboard/history.html (hunk 1 above)
Comment (groupsky) [reply rt_dashboard/templates/rt_dashboard/history.html]: The python data is inside the `{{ }}` and once processed it is turned into json and sent to browser. On the browser side it is first parsed as js then stringified to json string and back again to js. I don't see the benefit of the browser side processing.

Comment (vasilty) [reply rt_dashboard/templates/rt_dashboard/history.html]: Yeah, you are right, i fixed this

**Review by groupsky — APPROVED** [review]

**Review by i-trofimtschuk — APPROVED** [review]

**Pull request merged**